google/cloud/bigtable/table.py

```diff
@@ -1043,10 +1043,6 @@ class _RetryableMutateRowsWorker(object):
     are retryable, any subsequent call on this callable will be a no-op.
     """
 
-    RETRY_CODES = tuple(
```

Comment:
FYI, this will give `google/cloud/bigtable/table.py:1047: error: "None" has no attribute "value"` and even if you add an `if` that would validate non-none, it will persist. If we needed this const here (only used in one spot) we would have to likely do in a for loop, not comprehension. Possibly a shortcoming of mypy

Reply:
Could we appease it by adding `if retryable.grpc_status_code is not None`?